net-p2p/go-ipfs/go-ipfs-0.11.0.ebuild

```diff
@@ -1726,7 +1955,8 @@ src_test() {
 src_install() {
 	dobin ipfs
 	dobin ipfswatch
-	newbashcomp misc/completion/ipfs-completion.bash ipfs
+	./ipfs commands completion bash > ipfs-completion.bash || die
```

Comment:
Maybe move this line to  `src_compile()` as
```bash
./ipfs commands completion bash > misc/completion/ipfs-completion.bash || die
```

and leave this one untouched?

Maybe some make target is just missing which would create this file?

Reply:
In 0.10.0 upstream removed misc/completion folder entirely. If upstream won't use that folder to distribute other potential completions I don't see much value in creating a directory just to put that file tbh.

I will move that to src_compile, thanks.

Reply:
Ah, that's the reason. They probably outsourced that to package maintainers.